codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowControllerTest.java

```diff
@@ -1234,7 +1234,7 @@ private int streamableBytesForTree(Http2Stream stream) {
     }
 
     private Http2Stream stream(int streamId) throws Http2Exception {
```

Comment:
@nmittler - Did you mean to remove the `throws` here (and https://github.com/netty/netty/pull/3685#discussion_r28826700)?